src/main/java/com/rtb/projectmanagementtool/task/TaskSetStatusServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.rtb.projectmanagementtool.task;
+
+import com.google.appengine.api.datastore.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns task data */
+@WebServlet("/task-set-status")
+public class TaskSetStatusServlet extends HttpServlet {
+
+  DatastoreService datastore;
+
+  public TaskSetStatusServlet() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  // For testing only
+  public TaskSetStatusServlet(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get parameters
+    long taskID = Long.parseLong(request.getParameter("taskID"));
+    Status status = Status.valueOf(request.getParameter("status").trim().toUpperCase());
+
+    // Set status
+    TaskController taskController = new TaskController(datastore);
+    if (status == Status.COMPLETE) {
+      taskController.setComplete(taskID);
+    } else if (status == Status.INCOMPLETE) {
+      taskController.setIncomplete(taskID);
+    }
+  }
```

Comment:
What's the plan after calling this?  Would the page update the display?  How would the page know if changing the status was successful or not?

Reply:
I'm planning on reloading the task page info via the javascript function `getTaskInfo()` that's usually called in body onload. Maybe there's a more efficient way to only change the updated info, but for now I'm just going with the simplest option of fetching all of the task's info on each update.

Reply:
sgtm to go with simple approach first.

For partial loading, you can have the servlet return a json status code, then update the html page during fetch().then() when the response is received.  
eg. `if response.error=0 {document.getElementById('THE_TASK').setClassName('completed');}`